store/postgres/migrations/000006_create_lineage_graph_table.up.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
```

Comment:
same here, we don't need to start transaction here since we only have 1 operation in this migration file